Makefile

```diff
@@ -4,6 +4,8 @@ mockgen:
 	mockgen -source=pkg/openfeature/hooks.go -destination=pkg/openfeature/hooks_mock_test.go -package=openfeature
 	mockgen -source=pkg/openfeature/mutex.go -destination=pkg/openfeature/mutex_mock_test.go -package=openfeature
 test:
+	go test --short -cover ./...
+integration-test: # dependent on `docker run -p 8013:8013 ghcr.io/open-feature/flagd-testbed:latest`
```

Comment:
Not necessary, but in the Java and JS repos, I added the git submodule retrieval into the integration test goal/script. This has the benefit of "just working" when somebody runs it locally.

Reply:
Nice suggestion, done